test/infrastructure/docker/e2e/docker_suite_test.go

```diff
@@ -70,9 +78,15 @@ var _ = BeforeSuite(func() {
 	Expect(capiv1.AddToScheme(scheme)).To(Succeed())
 	Expect(bootstrapv1.AddToScheme(scheme)).To(Succeed())
 	Expect(infrav1.AddToScheme(scheme)).To(Succeed())
+	Expect(controlplanev1.AddToScheme(scheme)).To(Succeed())
+	Expect(cabpkv1.AddToScheme(scheme)).To(Succeed())
```

Comment:
We are already importing and adding this to the scheme with the `bootstrapv1` import.